src/DoctrineORMModule/Options/DBALConnection.php

```diff
@@ -4,10 +4,13 @@
 
 namespace DoctrineORMModule\Options;
 
-use Doctrine\DBAL\Driver\PDOMySql\Driver;
+use Doctrine\DBAL\Driver\PDO\MySQL\Driver as DriverV3;
+use Doctrine\DBAL\Driver\PDOMySql\Driver as DriverV2;
```

Comment:
I don't think that this is needed here. The class `Doctrine\DBAL\Driver\PDO\MySQL\Driver` is already available in DBAL 2 as well, see for https://github.com/doctrine/dbal/blob/2.13.x/lib/Doctrine/DBAL/Driver/PDO/MySQL/Driver.php. I think it has been added in 2.11 according to [this commit](https://github.com/doctrine/dbal/commit/67d6bb0395d3e3e1aadfd23f5658a2a846266dfd). So we should probably be save only using `PDO\MySQL`?

Reply:
I had problems with v2 and v3 compatibility, but in the meantime I up minimum version, so maybe now it works. To check

Reply:
As pointed out in https://github.com/doctrine/DoctrineORMModule/pull/675#issuecomment-960528042, raising the minimum dbal version brings in a forward compatibility layer in dbal 2.x and, hence, allows using the new symbols of 3.x in 2.x already.